### PR TITLE
Add Cultural Diversity Guideline to Remote Meeting

### PR DESCRIPTION
**Pull request by bigchickenwings**
- Adding a guideline focused on communication between different cultures and non-native speakers!

Please leave any comments/suggestions that you find relevant!

### CODE REVIEW

File: working-together/README.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
...
   - Be understanding of interruptions that may happen with remote work,
     e.g. children, pets, other people who might be sharing space with your teammate.
+- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
+  - **Avoid** whenever possible:
+    - Using slang when talking about an important subject
+    - Using passive voice (i.e. *"You will be asked some questions"*)
+    - Ambiguous phrases or words with multiple meanings
+    - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)
+  - **Prioritize** whenever possible:
+    - Short, direct sentences over long sentences
```
Comment (MatheusRich) [inline working-together/README.md:51]: I think we can link this article https://pudding.cool/2022/02/plain/ somewhere.

Also, feel free to pull more examples from it!

File: working-together/README.md [hunk 2]
```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
   - Consider calling in if you have an unreliable internet connection.
   - Be understanding of interruptions that may happen with remote work,
     e.g. children, pets, other people who might be sharing space with your teammate.
+- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
+  - **Avoid** whenever possible:
+    - Using slang when talking about an important subject
+    - Using passive voice (i.e. *"You will be asked some questions"*)
```
Comment (MatheusRich) [inline working-together/README.md:47]: I'm unsure why this is important. Could you clarify?

File: working-together/README.md (hunk 1 above)
Comment (bigchickenwings) [reply working-together/README.md:51]: Thank you! Gonna check it out!

File: working-together/README.md (hunk 2 above)
Comment (bigchickenwings) [reply working-together/README.md:47]: This is a recommendation from the [AAAL (American Association for Applied Linguistics)](https://www.aaal.org/guidelines-for-communication-rights##)!

Using passive voice can confuse non-native speakers as the wording can seem unusual depending on their mother language!

If we consider the example phrase, it would sound something like:
"You will be ask some questions", which could lead to a potential misunderstanding of "You will be asking some questions".

Active voice is something present in every language and normally what we are taught first in English, as well!

Comment (MatheusRich) [reply working-together/README.md:47]: That makes sense. Thanks for the clarification! I wonder if we could include that example in the text. It was very helpful!

Comment (bigchickenwings) [reply working-together/README.md:47]: I like that idea! Gonna try to add it :)

File: working-together/README.md [hunk 3]
```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
...
+    - Ambiguous phrases or words with multiple meanings
+    - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)
+  - **Prioritize** whenever possible:
+    - Short, direct sentences over long sentences
+    - Using simple words, as people are more used to them
+    - Using active voice (i.e. *"They will ask you some questions"*)
+    - Making sure that nobody feels guilty when they have difficulties expressing their thoughts
```
Comment (thoughtbot-summer) [inline working-together/README.md:54]: This also helps beyond cultural differences! For example people with stutters. However I might rephrase this as:

```suggestion
    - Making sure that others feel comfortable taking more time to express their thoughts
```

**Review by thoughtbot-summer — APPROVED** [review]
Love this!

I'd like to suggest some more considerations that would help autistic people in particular (may or may not count as a culture, depending on your perspective):

- Be understanding regarding whether others initiate or are comfortable participating in "small talk". If they do initiate it, and if you're comfortable, then it's encouraged to participate.
- Try to be aware of others' comfort level with eye contact; err on the side of less eye contact.
- For calls/videocalls, endeavor to use platforms that provide automatic closed captions.
- Favor being explicit rather than implicit with regards to expressing emotions or intent, especially in text-based communication (where emojis and tone tags can be helpful). When others' intent is not explicit, assume positive or neutral intent instead of negative intent.

(Of course feel free to adjust the phrasing on those to fit into "avoid"/"prioritize".)

File: working-together/README.md [hunk 4]
```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
   - Consider calling in if you have an unreliable internet connection.
   - Be understanding of interruptions that may happen with remote work,
     e.g. children, pets, other people who might be sharing space with your teammate.
+- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
+  - **Avoid** whenever possible:
+    - Using slang when talking about an important subject
+    - Using passive voice (i.e. *"You will be asked some questions"*)
+    - Ambiguous phrases or words with multiple meanings
+    - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)
```
[the remark this answers is not in the document]
Comment (bigchickenwings) [reply working-together/README.md:50]: I loved the way you phrased it! Is it okay if I use it this way?

[the remark this answers is not in the document]
Comment (thoughtbot-summer) [reply working-together/README.md:50]: Absolutely!

File: working-together/README.md (hunk 3 above)
Comment (bigchickenwings) [reply working-together/README.md:55]: Loved that as well!

**Comment by bigchickenwings** [comment]
@thoughtbot-summer I added most of your suggestions, except for eye contact, because I am changing the guidelines for the remote meeting!
But I believe we can and should open a PR to add this and maybe more stuff related to personal space (like avoiding touching people etc.) on the in-person meeting guidelines!

What do you think?

**Comment by thoughtbot-summer** [comment]
@bigchickenwings

> […] except for eye contact, because I am changing the guidelines for the remote meeting!

Makes sense!

> […] open a PR to add this and maybe more stuff related to personal space (like avoiding touching people etc.) on the in-person meeting guidelines!
> 
> What do you think?

Yeah I think that'd be great! And maybe we could extract many of the remote and in-person guidelines into a shared section, for example:

> ## Meeting Guidelines
> 
> - AAA
> - BBB
> - CCC
> 
> ### Remote Meetings
> 
> - DDD
> - EEE
> 
> ### In-Person Meetings
> 
> - FFF
> - GGG

**Comment by bigchickenwings** [comment]
@thoughtbot-summer Loved the idea of extracting, and I was thinking the same thing!
Do we need to get somebody's approval for this change, as it's something relatively big?

**Review by stefannibrasil — APPROVED** [review]
This is awesome!

File: working-together/README.md [hunk 5]
```diff
@@ -41,6 +41,36 @@ and valued during remote meetings that happen over video conferencing software.
...
+    - Taking a brief pause when changing subjects to make sure 
+      that the information isn’t getting mixed up.
+    - Be understanding regarding whether others initiate or are comfortable 
+      participating in "small talk". If they do initiate it, and if you're comfortable,
+      then it's encouraged to participate.
+    - Endeavor to use platforms that provide automatic closed captions.
+    - Developing awareness of when particular phrasing might be perceived by others
+      as harsh, and err on the side of being more flowery. Likewise, if others use language
+      that you perceive as harsh, consider that it might be a cultural difference;
+      but feel welcome to express if it makes you uncomfortable.
```
Comment (mike-burns) [inline working-together/README.md:70]: What about:

```suggestion
      that you perceive as harsh, consider that it might be a cultural difference, and consider asking for further clarification.
```

Comment (bigchickenwings) [reply working-together/README.md]: Just added this suggestion! Thank you Mike!

**Comment by sallyhall** [comment]
I really, really like these changes. Thank you for adding this @bigchickenwings! Thank you for including guides for autistic people, @thoughtbot-summer! I learned a lot from reading these suggestions.

**Comment by skybondsor** [comment]
Regarding this point:

> For calls/videocalls, endeavor to use platforms that provide automatic closed captions.

It would be helpful to list some examples of which platforms provide automatic closed captions. I, for one, do not know this list!

**Comment by thoughtbot-summer** [comment]
@skybondsor I'd love a list of that as well. :sweat_smile: I know that Google Meet has automation closed captions, but I don't think Slack does (at least when I last checked). I think Zoom has a closed captions feature, but I can't remember if it's automatic or whether it's necessary for someone to transcribe it in realtime, and I also can't remember if it requires admins to enable it (or gives admins the ability to disable it, both of which seem cruel). Phone calls of course do not have closed captions, unless you have some sort of specific setup for that. I'm looking for a list that we can link to, but I'm not finding anything that seems like it'll be kept up-to-date.

---

Side note: Google has an Android app called [Live Transcribe](https://www.android.com/accessibility/live-transcribe/) that can be useful for real-world interactions and platforms without closed captions (but of course that shouldn't be something we require).
